leapp.spec

```diff
@@ -0,0 +1,121 @@
+Name:       leapp
+Version:    0.0.1
+Release:    3
+Summary:    leapp tool rpm
+
+Group:      Unspecified
+License:    GPLv2+
```

Comment:
We should resolve https://github.com/leapp-to/prototype/issues/36 before merging this so we don't inadvertently indicate conflicting licensing terms anywhere.